source/common/ipc-api.js

```diff
@@ -33,3 +33,9 @@ export const GET_GPU_STATUS = {
   SUCCESS: `${GET_GPU_STATUS_CHANNEL}-success`,
   ERROR: `${GET_GPU_STATUS_CHANNEL}-error`,
 };
+
+const GET_DISK_SPACE_STATUS_CHANNEL = 'get-disk-space-status';
+export const GET_DISK_SPACE_STATUS = {
+  REQUEST: GET_DISK_SPACE_STATUS_CHANNEL,
+  SUCCESS: `${GET_DISK_SPACE_STATUS_CHANNEL}-success`,
+};
```

Comment:
Please refactor this dynamically typed constants-based (old) way of ipc channel communication to our new (3-4 months already) Flow-typed, promise based ipc channels: https://github.com/input-output-hk/daedalus/tree/develop/source/common/ipc 😉

Reply:
✅